src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -631,6 +637,37 @@ export default class DataTable extends Vue {
     return this.selectedRows.includes(row.rowId);
   }
 
+  _selectAllDropdown() {
+    const pageSelectionMessage = 'Select all items, this page';
+    const allPagesSelectionMessage = 'Select all items, all pages';
+    const deselectAllSelectionMessage = 'Deselect none';
```

Comment:
Sorry there was a typo in the design prototype. This should be 'Deselect all'
```suggestion
    const deselectAllSelectionMessage = 'Deselect all';
```